src/bundle/Resources/views/form_fields.html.twig

```diff
@@ -153,3 +153,18 @@
         {{ block('form_widget') }}
     </div>
 {%- endblock -%}
+
+{%- block richtext_widget -%}
+    {% set attr = attr|merge({ 'hidden': true }) %}
```

Comment:
How does this work, do we use `attr`?

Reply:
Yes, we do. Is used by the `widget_attributes` block called in https://github.com/ezsystems/ezplatform-admin-ui/pull/714/files#diff-c5ade6303c80a553058d3c44229e563bR165

Reply:
Great, thanks for explanation. :)